player/src/main/java/xyz/gianlu/librespot/player/Player.java

```diff
@@ -382,6 +382,7 @@ public void finishedLoading(@NotNull MetadataWrapper metadata) {
             public void loadingError(@NotNull Exception ex) {
                 if (ex instanceof PlayableContentFeeder.ContentRestrictedException) {
                     LOGGER.error("Can't load track (content restricted).", ex);
+                    events.playbackFailed(ex);
```

Comment:
While we're at it I think we could move this up a couple of lines and call it if any error happens.

Reply:
Sure, but then if the exception is not ContentRestrictedException, onPanicState() will also be called. Is that okay? We would then call onPlaybackFailed() first, then onPanicState().

Reply:
Done!